rest-api-spec/src/main/resources/rest-api-spec/test/indices.rollover/40_mapping.yml

```diff
@@ -0,0 +1,45 @@
+---
+"Typeless mapping":
+  - skip:
+      version: " - 6.99.99"
+      reason:  include_type_name defaults to true before 7.0.0
+
+  - do:
+      indices.create:
+        index: logs-1
+        body:
+          aliases:
+            logs_search: {}
+
+  # index first document and wait for refresh
+  - do:
+      index:
+        index: logs-1
+        type:  test
```

Comment:
I'm not sure it's worth another update + build, but a note that we've been trying to avoid typed index calls (unless we're explicitly testing typed behavior as in `41_mapping_with_types`).